regression_check.sh

```diff
@@ -11,7 +11,7 @@ POLARIZED_UNPOLPDF_NAME="NNPDF40_nlo_pch_as_01180"
 
 LIST_DIS_DATASETS=(
   "HERA_CC_318GEV_EP-SIGMARED"
-  "NNPDF_POS_2P24GEV_F2D"
+  # "NNPDF_POS_2P24GEV_F2D"
```

Comment:
@cschwan, you're right. The results are the same in both versions using the grid and EKO I provided you before. Somehow, that EKO and the ones I have locally and the in the benchmarks were not the same. But this is now fixed.

Now, we can make a new release that includes the fixes to empty grids, add back the POS dataset in the regression, and this PR should be done.
```suggestion
  "NNPDF_POS_2P24GEV_F2D"
```

Reply:
Addressed in [69268eb](https://github.com/NNPDF/pineko/pull/206/commits/69268ebe5309aae91afb5abc1c276be6393dc93d).